lib/rihanna/job.ex

```diff
@@ -177,6 +177,65 @@ defmodule Rihanna.Job do
   end
 
   @doc false
+  def delete_by(opts) do
```

Comment:
There's a lot of code in here, I'm wondering if `Rihanna.Job` is getting a bit overloaded.

@lpil related to our whatsapp discussion the other day, what do you think about moving these to a `Rihanna.DatabaseOps` module or somethign similar?

Reply:
Sure. Would enable us to use ecto there too.

Reply:
Do you think this should be a separate library?

Reply:
That or it checks if ecto is loaded before generating the ecto code.

I think I would prefer we migrate to strings and do that rather than merge this one